### DISCUSSION TITLE
群晖docker挂载目录读取权限问题

### DISCUSSION THREAD

**Discussion by TwoBall** [kkkgo/PaoPaoDNS#52] [问答和帮助]
最近一次清洁重装后一直不生成缓存；
还有一个问题就是在群晖上http file能启动，但是我拖进去的ppgw.ini因为拥有者不是root而读取不到。之前的版本好像都没有问题。

**Comment by kkkgo** [comment]
4天前的版本没有代码变化，仅Actions自动定期构建整合最新数据。   
redis.rdb需要满足最小体积才会生成。或者你可以挂PaoPao-pref。参考 https://github.com/kkkgo/PaoPaoDNS/discussions/12  

如果你的http文件读取不了，那可能需要考虑`/data`目录对容器是否有读写权限？如果没有，`redis.rdb`自然也写不进去。

**Comment by kkkgo** [comment]
参考：https://www.orcy.net.cn/1636.html   
手上没有群晖，你可以尝试探索下再反馈。

**Reply by kkkgo** [reply]
![image](https://github.com/kkkgo/PaoPaoDNS/assets/4530897/b608116a-5dbd-4727-8a31-8c432786f98b)
![image](https://github.com/kkkgo/PaoPaoDNS/assets/4530897/3c68a5aa-61f0-4c1c-8629-265d574be0fc)
![image](https://github.com/kkkgo/PaoPaoDNS/assets/4530897/c30bee65-1b9f-4192-89c2-9f228548db27)
![image](https://github.com/kkkgo/PaoPaoDNS/assets/4530897/96a58a31-1e36-42e1-b586-615db2a8c973)
![image](https://github.com/kkkgo/PaoPaoDNS/assets/4530897/2a38933f-4e39-4886-8159-11477ad933f8)

**Reply by TwoBall** [reply]
好的，我测试一下再反馈
